docs/dev/check_references.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Integration assets reference
+kind: documentation
+---
+
+## Configuration file
+
+When preparing a new integration, you must include an example configuration that contains the necessary options and reasonable defaults. The example configuration file, which in this case is located at `<CHECK_NAME>/datadog_checks/<CHECK_NAME>/data/conf.yaml.example`, has two top-level elements: `init_config` and `instances`. The configuration under `init_config` is applied to the integration globally, and is used in every instantiation of the integration, whereas anything within `instances` is specific to a given instantiation.
```

Comment:
The impetus for me creating the card (as stated in its description) was the introduction of https://datadoghq.dev/integrations-core/meta/config_specs/

All new integrations use those

Reply:
Yup indeed, this is the first step as we needed to split the "reference" material from the how to guide. Once done we will update the configuration file instructions to point to this new knowledge of conf file templating.